src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -1,6 +1,7 @@
 //
 // Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Licensed under the MIT license. See LICENSE file in the project root for full
+// license information.
```

Comment:
I checked in `.clang-format` and `clang-format-3.6 -dump-config` shows a column limit of 120.  I will try to figure out why this is happening.